README.md

```diff
@@ -1,9 +1,53 @@
-# ml4ir: Machine Learning libraries for Information Retrieval
+# ml4ir: Machine Learning for Information Retrieval
+**Quickstart &rarr;** **[ml4ir Read the Docs](https://ml4ir.readthedocs.io/en/latest/)** | **[ml4ir pypi](https://pypi.org/project/ml4ir/)** | **[python ReadMe](python/)**
 
-## Getting Started
 
-[Quickstart](python/README.md)
+ml4ir is an open source library for training and deploying deep learning models for search applications. ml4ir is built on top of **python3** and **tensorflow 2.x** for training and evaluation. It also comes packaged with scala utilties for **JVM inference**.
+
+ml4ir is designed as modular subcomponents which can each be combined and customized to build a variety of search ML models such as:
+* Learning to Rank
+* Query Auto Completion
+* Document Classification
+* Query Classification
+* Named Entity Recognition
+* Top Results
+* Query2SQL
+* *add your application here*
+  
+![ml4ir](python/docs/source/_static/ml4ir.png)
+
+
+## Motivation
+Search is a complex data space with lots of different types of ML tasks working on a combination of structured and unstructured data sources. There is no single library that
+* provides an end-to-end training and serving solution for a variety of search applications
+* allows training of models with limited coding expertise
+* allows easy customization to build complex models to tackle the search domain
+* focuses on performance and robustness
+* enables fast prototyping
+
+So, we built ml4ir to do all of the above. 
+
+### Guiding Principles
+**Customizable Library**
+
+First and foremost, we want ml4ir to be an easy-to-use and highly customizable library so that you can build the search application of your need. ml4ir allows each of its subcomponents to be over-riden, mixed and match with other custom modules to create and deploy models.
```

Comment:
"First and foremost" is one of my favorite business buzzword phrases to make fun of ;) I think this sentence would be clearer as "We built ml4ir with the primary goal to be" or "We designed ml4ir to be" or "We want ml4ir to be" or "Our primary goal was to build ml4ir to be". 

Also "of your need" sounds odd. Could say "that you need" or "of your choice"

Reply:
Also "overridden" not "over-riden"